macaroonbakery/tests/test_bakery.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2016 Canonical Ltd.
+# Licensed under the AGPLv3, see LICENCE file for details.
+import base64
+import json
+
+from unittest import TestCase
```

Comment:
unittest is in the stdlib, please move it to the first import block. Both mock and httpmock are in the second import block. Sorry, Brad would blame my obsessive compulsive disorder.